vsts/pipelines/nightly.yml

```diff
@@ -61,7 +61,74 @@ jobs:
       echo "##vso[task.setvariable variable=BuildRuntimeImages;]false"
       echo "##vso[task.setvariable variable=TestBuildImages;]false"
       echo "##vso[task.setvariable variable=TestRuntimeImages;]false"
-      echo "##vso[task.setvariable variable=TestIntegrationCaseFilter;]Category!=DB"
+      echo "##vso[task.setvariable variable=TestIntegrationCaseFilter;]Category=PYTHON"
+      echo "##vso[task.setvariable variable=TestIntegration;]true"
+      echo "##vso[task.setvariable variable=PushBuildImages;]false"
+      echo "##vso[task.setvariable variable=PushRuntimeImages;]false"
+      echo "##vso[task.setvariable variable=PushToDockerHub;]false"
+      echo "##vso[task.setvariable variable=EmbedBuildContextInImages;]false"
+    displayName: 'Set variables'
+  - template: _buildParallel.yml
+
+- job: Job_DotNetCoreIntegrationTests
+  displayName: Test DotNetCore Integration Tests
+  dependsOn: 
+    - Job_BuildImage
+    - Job_RuntimeImages
+  pool:
+    name: OryxLinux
+  steps:
+  - script: |
+      echo "##vso[task.setvariable variable=BuildBuildImages;]false"
+      echo "##vso[task.setvariable variable=BuildRuntimeImages;]false"
+      echo "##vso[task.setvariable variable=TestBuildImages;]false"
+      echo "##vso[task.setvariable variable=TestRuntimeImages;]false"
+      echo "##vso[task.setvariable variable=TestIntegrationCaseFilter;]Category=DOTNETCORE"
+      echo "##vso[task.setvariable variable=TestIntegration;]true"
+      echo "##vso[task.setvariable variable=PushBuildImages;]false"
+      echo "##vso[task.setvariable variable=PushRuntimeImages;]false"
+      echo "##vso[task.setvariable variable=PushToDockerHub;]false"
+      echo "##vso[task.setvariable variable=EmbedBuildContextInImages;]false"
+    displayName: 'Set variables'
+  - template: _buildParallel.yml
+
+- job: Job_NodeIntegrationTests
+  displayName: Test NodeJs Integration Tests
+  dependsOn: 
+    - Job_BuildImage
+    - Job_RuntimeImages
+  pool:
+    name: OryxLinux
+  timeoutInMinutes: 150
```

Comment:
Seems like a lot - the others didn't get such a high timeout

Reply:
yeah node integration tests sometimes taking more than 45 minutes